pythclient/pythclient.py

```diff
@@ -13,18 +13,18 @@
 from .solana import SolanaAccount, SolanaClient, SolanaPublicKey, SOLANA_DEVNET_HTTP_ENDPOINT, SOLANA_DEVNET_WS_ENDPOINT, SolanaPublicKeyOrStr
 from .pythaccounts import PythAccount, PythMappingAccount, PythProductAccount, PythPriceAccount
 from . import exceptions, config, ratelimit
+from .utils import get_key
 
-V1_FIRST_MAPPING_ACCOUNT_KEY = "ArppEFcsybCLE8CRtQJLQ9tLv2peGmQoKWFuiUWm4KBP"
-V2_FIRST_MAPPING_ACCOUNT_KEY = "BmA9Z6FjioHJPpjT39QazZyhDRUdZy2ezwx4GiDdE2u2"
-V2_PROGRAM_KEY = "gSbePebfvPy7tRqimPoVecS2UsBvYv46ynrzWocc92s"
+V2_FIRST_MAPPING_ACCOUNT_KEY = get_key("devnet", "mapping")
+V2_PROGRAM_KEY = get_key("devnet", "program")
```

Comment:
Do we want to have the side effect like this at import time? Personally, I'd be ok with leaving `first_mapping_account_key` in the `PythClient` class blank and requiring it. Then we can remove these as constants, but make sure to put examples of them elsewhere.

wdyt?